package.json

```
@@ -53,7 +53,7 @@
     "typescript": "^4.1.3"
   },
   "peerDependencies": {
-    "class-transformer": "^0.2.3",
+    "class-transformer": "0.3.1",
```

```suggestion
    "class-transformer": "0.2.3 - 0.3.1",
```

How about this to still allow the previously allowed versions?

---

Just imo: I prefer not leaving people with old versions of dependencies, if I don't have to.

After looking at the class-transformer changelog and version history: There is no version between 0.2.3 and 0.3.1.
Therefore locking it to 0.3.1 would just be a dependency bump to the "next" version.

If you insist on allowing the previous version, I'll comply - it's just not my style :)

Just for reference:
* Class Transformer Tags: https://github.com/typestack/class-transformer/tags
* Class Transformer Changelog: https://github.com/typestack/class-transformer/blob/develop/CHANGELOG.md

---

My thinking is that since this library works fine with `class-transformer@0.2.3` there's no reason to throw `incompatible peer dependency` errors to folks using that version: it's up to them to upgrade or not.

If `class-transformer` was a direct (non-peer) dependency here then we'd bump the version for sure.